azure-pipelines.yml

```diff
@@ -28,22 +28,22 @@ strategy:
     Linux GCC Native DLL:
       VM_IMAGE: 'ubuntu-20.04'
       INSTALL_DEPS: 'sudo apt-get update && sudo apt-get -y -q --no-install-recommends install libfreetype6-dev liblua5.2-dev python-yaml python-jinja2 ninja-build gcc-8 g++-8'
-      CMAKE_OPTIONS: '-G Ninja -DDAEMON_CBSE_PYTHON_PATH=/usr/bin/python2 -DCMAKE_C_COMPILER=gcc-8 -DCMAKE_CXX_COMPILER=g++-8 -DBUILD_GAME_NATIVE_DLL=1 -DBUILD_GAME_NATIVE_EXE=0 -DBUILD_GAME_NACL=0'
+      CMAKE_OPTIONS: '-G Ninja -DCBSE_PYTHON_PATH=/usr/bin/python2 -DCMAKE_C_COMPILER=gcc-8 -DCMAKE_CXX_COMPILER=g++-8 -DBUILD_GAME_NATIVE_DLL=1 -DBUILD_GAME_NATIVE_EXE=0 -DBUILD_GAME_NACL=0'
       BUILDER_OPTIONS: '-j$(nproc) -k 10'
     Linux Clang Native DLL:
       VM_IMAGE: 'ubuntu-20.04'
       INSTALL_DEPS: 'sudo apt-get update && sudo apt-get -y -q --no-install-recommends install libfreetype6-dev liblua5.2-dev python-yaml python-jinja2 ninja-build'
-      CMAKE_OPTIONS: '-G Ninja -DDAEMON_CBSE_PYTHON_PATH=/usr/bin/python2 -DCMAKE_C_COMPILER=clang -DCMAKE_CXX_COMPILER=clang++ -DBUILD_GAME_NATIVE_DLL=1 -DBUILD_GAME_NATIVE_EXE=0 -DBUILD_GAME_NACL=0'
+      CMAKE_OPTIONS: '-G Ninja -DCBSE_PYTHON_PATH=/usr/bin/python2 -DCMAKE_C_COMPILER=clang -DCMAKE_CXX_COMPILER=clang++ -DBUILD_GAME_NATIVE_DLL=1 -DBUILD_GAME_NATIVE_EXE=0 -DBUILD_GAME_NACL=0'
```

Comment:
Python2 is deprecated, unsupported, and dead. We should not test it and should not care if we break it.